CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# How to contribute
+
+Contributions to ScalaBridge are welcomed and encouraged. Without input from volunteers we would have nothing. By participating in this project you agree to abide by the [Bridge Foundry](http://bridgefoundry.org/) [code of conduct](http://bridgefoundry.org/code-of-conduct/).
```

Comment:
Let's sound more excited!!

`Contributions to ScalaBridge are welcomed and encouraged. Without input from volunteers we would have nothing.` 

to

`Contributions to ScalaBridge are welcomed and encouraged - without input from volunteers we would have nothing!`